wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCellDescription.swift

```diff
@@ -53,22 +53,16 @@ final class ConversationSystemMessageCellDescription {
             return [AnyConversationMessageCellDescription(renamedCell)]
 
         case .missedCall:
-            let missedCallCell = ConversationCallSystemMessageCellDescription(
+            let missedCallCell = ConversationMissedCallSystemMessageCellDescription(
                 message: message,
-                data: systemMessageData,
-                missed: true
+                data: systemMessageData
             )
 
             return [AnyConversationMessageCellDescription(missedCallCell)]
 
         case .performedCall:
-            let callCell = ConversationCallSystemMessageCellDescription(
-                message: message,
-                data: systemMessageData,
-                missed: false
-            )
-
-            return [AnyConversationMessageCellDescription(callCell)]
+            // [WPB-6988] removed system message for call ends.
+            return []
```

Comment:
So I guess this means that for any existing performed call system messages we just don't show them in the conversations anymore?

Reply:
Yes that's how it should work. Even if you restore from a backup the view should not create any cells.

Reply:
Maybe we should mark the system message type as deprecated or remove it completely (without breaking).

Reply:
Good point, it needs to be used in a few places but is marked as deprecated now 45bf327bc63b884161ccc056a07a5f98f20f3009.